torch/onnx/onnx_supported_ops.py

```diff
@@ -0,0 +1,122 @@
+from inspect import signature, _empty  # type: ignore[attr-defined]
+from torch.onnx.symbolic_registry import _registry, register_version
+from typing import List, Dict
+
+
+# Class for Torch Schema
+class TorchSchema:
+    name: str
+    overload_name: str
+    arguments: List[str]
+    optional_arguments: List[str]
+    returns: List[str]
+    opsets: List[int]
+
+    def __init__(self, schema, symbolic=False) -> None:
+        if not symbolic:
+            self.name = schema.name
+            self.overload_name = schema.overload_name
+            self.arguments = [arg.name for arg in schema.arguments]
+            self.optional_arguments = []
+            self.returns = [ret.name for ret in schema.returns]
+            self.opsets = []
+        else:
+            self.name = schema
+            self.overload_name = ""
+            self.arguments = []
+            self.optional_arguments = []
+            self.returns = []
+            self.opsets = []
+
+
+    def __str__(self) -> str:
+        s = f"{self.name}.{self.overload_name}("
+        s += ", ".join(self.arguments)
+        s += ") -> ("
+        s += ", ".join(self.returns)
+        s += ")"
+        s += " in opsets "
+        s += ", ".join(str(opset) for opset in self.opsets)
+        return s
+
+    def __eq__(self, other) -> bool:
+        if not isinstance(other, TorchSchema):
+            return False
+        if self.name == other.name:
+            # TODO: Handle overloads
+            return True
+        return False
+
+    def is_aten(self) -> bool:
+        return self.name[:6] == "aten::"
+
+    def is_backward(self) -> bool:
+        return "backward" in self.name
+
+
+# Create TorchSchema object directory of all aten schemas
+def get_all_aten_forward_schemas():
```

Comment:
for all the functions in this file:
remove `get_` prefix. The name will be just as clear but shorter without it.